src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="form form--big"
+      action="search"
+      data-qa="big"
+    >
+      <input
+        class="input form--big__input"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
```

Comment:
The attributes for the `input` tag should start on a new line with a 2-space indentation relative to the tag to keep the attributes correctly formatted.